lib/index.js

```diff
@@ -104,7 +104,9 @@ const setup = ({customTextFormattings, parseFigureProps, customCaption} = {}) =>
       assert(oldArticleElm, 'oldArticleElm must exists');
       assert(newArticleElm, 'newArticleElm must exists');
 
-      patchDom({oldArticleElm, newArticleElm});
+      if (oldArticleElm.innerHTML !== newArticleElm.innerHTML) {
```

Comment:
ah good point, I just added it because I thought it might be faster, but I'll make sure that's true before merging. thanks!